cdap-ui/app/cdap/components/OpsDashboard/RunsGraph/DataParser.js

```diff
@@ -100,10 +90,6 @@ export function parseDashboardData(rawData, startTime, duration, pipeline, custo
     }
   });
 
-  timeArray.forEach((time) => {
-    buckets[time].runsList = uniqBy(buckets[time].runsList, 'run');
-  });
-
```

Comment:
if you are readding the pushing of run to runsList for schedule, then you need this uniqBy again